cinnabar/stats.py

```diff
@@ -14,6 +14,8 @@ def bootstrap_statistic(
     statistic: str = "RMSE",
     nbootstrap: int = 1000,
     plot_type: str = "dG",
+    bootstrap_experimental_uncertainty: bool = False,
```

Comment:
I would prefer we use the labels `x_true` and `y_pred` for these in the same manner as the other arguments for the method. Whilst the normal use is experiment & predicted, there is no reason why this can't be predicted1 vs predicted2, or even experiment1 vs experiment2.

Reply:
Fixed!

Reply:
I agree with this suggestion, it is better if we have names that tell more. Maybe something that  has `true` and `predicted` on the names, `include_true_errors` and `include_predicted_errors` or something similar. By the name of the function we already know this is related to boostrapping, so it feels unnecessary to have it in the `kwarg` name.

Reply:
Just changed `bootstrap` to `include` for the arguments to `bootstrap_statistics`. Did not change the argument names plotting.py and plotlying.py functions